### PR TITLE
create static pages

### PR DESCRIPTION
**Pull request by DucNguyenVan**
apply I18n

### CODE REVIEW

File: app/assets/stylesheets/custom.css.scss [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+  letter-spacing: -1px;
+  padding-top: 9px;
+  font-weight: bold;
+}
+
+#logo:hover {
+  color: #fff;
+  text-decoration: none;
+}
+
```
Comment (ThuBM) [inline app/assets/stylesheets/custom.css.scss]: Xoa line nay

File: app/assets/stylesheets/custom.css.scss [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+footer ul {
+  float: right;
+  list-style: none;
+}
+
+footer ul li {
+  float: left;
+  margin-left: 15px;
+}
+
```
Comment (ThuBM) [inline app/assets/stylesheets/custom.css.scss]: xoa line

File: app/assets/stylesheets/custom.css.scss [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+/* mixins, variables, etc. */
+
+$gray-medium-light: #eaeaea;
+
+@mixin box_sizing {
+  -moz-box-sizing:    border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing:         border-box;
+}
+
```
Comment (ThuBM) [inline app/assets/stylesheets/custom.css.scss]: xoa

File: app/views/layouts/_header.html.erb [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+              <li>
+                <%= link_to t("logout") %>
+              </li>
+            </ul>
+        </li>
+        <li><%= link_to t("help"),   help_path %></li>
+      </ul>
+    </nav>
+  </div>
+</header>
```
Comment (ThuBM) [inline app/views/layouts/_header.html.erb]: reindent lai file nay

File: app/helpers/application_helper.rb [hunk 5]
```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+    def full_title(page_title = '')
```
Comment (phonghenry14) [inline app/helpers/application_helper.rb]: remove ()

File: app/views/layouts/application.html.erb [hunk 6]
```diff
@@ -1,14 +1,20 @@
 <!DOCTYPE html>
```
Comment (phonghenry14) [inline app/views/layouts/application.html.erb]: replace all ' with "

File: app/views/static_pages/home.html.erb [hunk 7]
```diff
@@ -0,0 +1,6 @@
+<% provide(:title, t("home")) %>
```
Comment (phonghenry14) [inline app/views/static_pages/home.html.erb]: remove ()

File: app/views/layouts/application.html.erb [hunk 8]
```diff
@@ -1,14 +1,20 @@
...
 <html>
 <head>
-  <title>BaseApp</title>
+  <title><%= full_title yield :title %></title>
```
Comment (phonghenry14) [inline app/views/layouts/application.html.erb]: add ()

File: config/routes.rb [hunk 9]
```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
```
Comment (phonghenry14) [inline config/routes.rb]: replace all ' with "

File: app/views/static_pages/help.html.erb [hunk 10]
```diff
@@ -0,0 +1,6 @@
+<% provide :title, t("help") %>
+
+<h1>Help</h1>
```
Comment (phonghenry14) [inline app/views/static_pages/help.html.erb]: i18n

File: app/helpers/static_pages_helper.rb [hunk 11]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```
Comment (dieunb) [inline app/helpers/static_pages_helper.rb]: remove this file

**Pull request merged**